### ISSUE TITLE
Evaluating pretraining results

### ISSUE THREAD

**Issue by usuyama** [microsoft/AzureML-BERT#46]
Would be great to get expected pretraining results (e.g., loss, next-sentence-prediction accuracy, etc.) and learning curves using the 64 * V100 training.

I found the fine-tuning task results on the [blog](https://azure.microsoft.com/en-us/blog/microsoft-makes-it-easier-to-build-popular-language-representation-model-bert-at-large-scale/) but would be nice to compare the pretraining results as a "sanity check."

I don't have access to V100 GPUs for now so I'm using P40 GPUs to replicate/validate the pretraining pipeline.

@maxluk @aashnamsft any pretraining details that you guys can share? Even a screenshot of the AzureML metrics page would be helpful to compare and double-check.
